src/components/WorldMap/index.js

```diff
@@ -165,11 +188,25 @@ class WorldMap extends Component {
           <h2><span>9</span> war crimes <span>75,000</span> victims</h2>
           <p>The crimes on the map have been committed between 2004 and now. Start exploring the map and find out which crimes are being investigated by the International Criminal Court.</p>
         </header>
+
+        <div className={styles.closestCase}>
+          <p>Closest war crime to you:</p>
+          {this.state.userLocation ? <h3>{this.state.closestWarcrime}</h3> : <div className={styles.isLoading}></div>}
+
+          <Link
+            className={this.state.userLocation ? `${styles.button} ${styles.buttonRed}` : styles.button}
+            to="selectedCase"
+            spy={true}
+            smooth={true}
+            duration={500}
+          >Learn more</Link>
+        </div>
+
         <div className={styles.WorldMapContent}>
           { featureCollection &&
             <svg
               className={styles.WorldMapSVG}
-              viewBox="0 0 961 620" preserveAspectRatio="xMidYMin slice"
+              viewBox="0 0 941 600" preserveAspectRatio="xMidYMin slice"
```

Comment:
why you change this? de path functie van d3 die gebruikt wordt om de map te tekenen rekent op die maten.